BDPointAndroidXamarinDemo/BDPointAndroidXamarinDemo.csproj

```diff
@@ -98,17 +94,21 @@
       <SubType></SubType>
       <Generator></Generator>
     </AndroidResource>
+    <AndroidResource Include="Resources\values\dimen.xml">
+      <SubType></SubType>
+      <Generator></Generator>
+    </AndroidResource>
   </ItemGroup>
   <ItemGroup>
     <Folder Include="Resources\drawable\" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.4.0" />
+    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.7.0.1" />
     <PackageReference Include="Xamarin.AndroidX.Work.Work.Runtime.Ktx">
-      <Version>2.5.0.1</Version>
+      <Version>2.7.1.5</Version>
     </PackageReference>
     <PackageReference Include="Bluedot.PointSDK.Android">
-      <Version>15.4.1</Version>
+      <Version>15.5.1.2-alpha</Version>
```

Comment:
This is currently using pre-release SDK version 15.5.1.2-alpha. Once it passed testing, and version 15.5.1 is release, I will update it to point to 15.5.1. Ticket created [here](https://bluedotinnovation.atlassian.net/browse/BD-4505?atlOrigin=eyJpIjoiMGZlYzg4OTVjYzdiNGNjNjgzYTE5ZDU2ZWMxN2IyY2QiLCJwIjoiaiJ9)

Reply:
Makes sense that Min App is using a prerelease package 15.5.1.2-alpha for testing 👍